zone/client_packet.cpp

```diff
@@ -4276,8 +4276,7 @@ void Client::Handle_OP_Bug(const EQApplicationPacket *app)
 
 void Client::Handle_OP_Camp(const EQApplicationPacket *app)
 {
-	if (ClientVersion() == EQ::versions::ClientVersion::RoF2 && RuleB (Parcel, EnableParcelMerchants) &&
-		GetEngagedWithParcelMerchant()) {
```

Comment:
Why was this removed? Only thing I noticed that could be changed was pushing `RuleB` up to the `(` like `RuleB(`.

Reply:
My mistake.  Misunderstood.  Updated.